pkg/defkinds/php/definition.go

```diff
@@ -595,83 +578,68 @@ func addIntegrations(stageDef *StageDefinition) error {
 //
 // This list has been obtained using:
 // docker run --rm -t php:7.2-fpm-buster php -r 'var_dump(get_loaded_extensions());'
-var preinstalledExtensions = []string{
```

Comment:
what about also update the above comment to indicate what the struct is supposed to contain ?

Reply:
Mh I can but using a map with empty struct/interface as value is quite a usual pattern in golang whenever a slice starts to be too big :wink: